packages/remix-cloudflare-pages/worker.ts

```diff
@@ -0,0 +1,96 @@
+import type { ServerBuild, AppLoadContext } from "@remix-run/server-runtime";
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/server-runtime";
+
+export interface CreateRequestHandlerParams<Env = any> {
+  build: ServerBuild;
+  getLoadContext?: (context: EventContext<Env, any, any>) => AppLoadContext;
+  mode?: string;
+}
+
+function createInternalHandler<Env = any>({
```

Comment:
Y'all call this `createRequestHandler` in the `@remix-run/cloudflare-workers` package.